eng/scripts/vs.buildtools.json

```diff
@@ -17,8 +17,6 @@
         "Microsoft.VisualStudio.Component.NuGet.BuildTools",
         "Microsoft.VisualStudio.Component.VC.ATL",
         "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
-        "Microsoft.VisualStudio.Component.VC.v141.ATL",
```

Comment:
IIRC there wasn't a v142 equivalent for this package. As long as things build correct, I think we are fine to remove this.

Reply:
"Microsoft.VisualStudio.Component.VC.ATL" is the v142 equivalent, and it's listed already. :)

Reply:
Yep, I came to the same conclusion looking at the component directory: https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-community?view=vs-2019